docs/openstack/backend_services_deployment.md

```diff
@@ -140,7 +140,8 @@ podified OpenStack control plane services.
 
     ovs:
       enabled: false
-      template: {}
+      template:
+        external-ids: {}
```

Comment:
i was just digging into this as well

i found the spec there https://github.com/openstack-k8s-operators/openstack-operator/blob/d54ddb6dde0d93f7c2da3a0055dcc1ccc95f1e8f/apis/bases/core.openstack.org_openstackcontrolplanes.yaml#L8678

so this seems reasonable